cjs/index.d.ts

```diff
@@ -0,0 +1,3 @@
+declare const globaThis: any;
+export default globaThis;
```

Comment:
[This is incorrect.](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require)